x/ecocredit/client/tx.go

```diff
@@ -49,15 +47,16 @@ func txflags(cmd *cobra.Command) *cobra.Command {
 
 func txCreateClass() *cobra.Command {
 	return &cobra.Command{
-		Use:   "create-class [designer] [issuer[,issuer]*] [metadata]",
+		Use:   "create-class [designer] [issuer[,issuer]*] [metadata] [credit type]",
```

Comment:
```suggestion
		Use:   "create-class [designer] [issuer[,issuer]*] [credit type] [metadata]",
```

Not sure why exactly, but I like to have `metadata` at the end, once the regular data is done